pkg/test/resourcefixture/testdata/basic/certificatemanager/v1beta1/certificatemanagercertificate/certificatemanagercertificateselfmanaged/_generated_object_certificatemanagercertificateselfmanaged.golden.yaml

```diff
@@ -34,7 +20,6 @@ spec:
   projectRef:
     external: ${projectId}
   resourceID: certificatemanagercertificate${uniqueId}
-  scope: DEFAULT
```

Comment:
Odd that this one disappears ... will be interesting to cross-check against real-gcp!